Marlin/configuration_store.cpp

```diff
@@ -555,6 +555,7 @@ void Config_ResetDefault() {
 
   #if ENABLED(AUTO_BED_LEVELING_FEATURE)
     zprobe_zoffset = Z_PROBE_OFFSET_FROM_EXTRUDER;
+    update_software_endstops(Z_AXIS);
   #endif
```

Comment:
If the `zprobe_zoffset` matters to the `sw_endstop_min[Z_AXIS]` then the call to `update_software_endstops(Z_AXIS)` is necessary to apply it here. Only the `sw_endstop_min[Z_AXIS]` will be affected by this call, as nothing else will have changed.